fortls/objects.py

```diff
@@ -1318,6 +1326,19 @@ def get_actions(self, sline, eline):
             ]
         return actions
 
+    def get_hover(self, long=False, include_doc=True, drop_arg=-1):
+        keywords = ""
+        if self.abstract:
+            keywords += ", ABSTRACT"
+        if self.inherit:
+            keywords += ", EXTENDS({})".format(self.inherit)
+        hover = "TYPE{} :: {}".format(keywords, self.name)
+        if include_doc:
+            doc_str = self.get_documentation()
+            if doc_str is not None:
+                hover += "\n" + doc_str
+        return hover, long
```

Comment:
use fstrings and consider using a list for keywords.
I think that explicit newline character might not be correct now that Markdown is being used for hover messages.

 Also, we need to provide some unit tests for the non-covered lines.